packages/lesswrong/components/notifications/NotificationsMenu.tsx

```diff
@@ -74,10 +74,6 @@ const NotificationsMenu = ({ classes, open, setIsOpen, hasOpened }: {
 }) => {
   const currentUser = useCurrentUser();
   const [tab,setTab] = useState(0);
-  const [notificationTerms,setNotificationTerms] = useState<NotificationsViewTerms>({view: 'userNotifications'});
-  const [lastNotificationsCheck,setLastNotificationsCheck] = useState(
-    (currentUser?.lastNotificationsCheck) || ""
-  );
```

Comment:
`notificationTerms` is just a function of `tab`, so it doesn't need to be an additional piece of react state. It's now set below.

Similarly, `lastNotificationsCheck` doesn't change at all -- there were no calls to `setLastNotificationsCheck`.

Reply:
This change is incorrect. While it's true that `setLastNotificationsCheck` is never called, removing the `useState()` wrapper is still a significant change because `currentUser?.lastNotificationsCheck` changes when the drawer opens. This change causes the little number next to the envelope icon (which tells you you have new private messages) to disappear a fraction of a second after the notifications menu is opened.